terraform/files/manager-part-1.yml

```diff
@@ -8,7 +8,7 @@
     ansible_python_interpreter: /usr/bin/python3
     configuration_git_host: github.com
     configuration_git_port: 443
-    configuration_git_repository: osism/testbed.git
```

Comment:
This should not be committed. I think it makes sense to be able to configure this via the Makefile of the testbed. At the moment it is only possible to change the used branch.

Reply:
I have changed this back to the upstream.